### PR TITLE
docs: add opengraph tags and image

### LINKED ISSUE #994: docs: add opengraph meta tags
The docs are missing opengraph tags such as image, title, description, etc.

### PR DESCRIPTION
**Pull request by markmichon**
## Description
Adds opengraph tags and image to doc pages

## Related
- Close #994 

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.
- [x] PR title follows [Conventional Commits](https://www.conventionalcommits.org/) format

### CODE REVIEW

File: docs/_src/_includes/layouts/base.njk [hunk 1]
```diff
@@ -7,6 +7,13 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge"/>
     <meta name="google-site-verification" content="TPIn3qa-ZOQsCQrpOB6J5eALE4dPBPiRBEJP5azpo9g"/>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, viewport-fit=cover"/>
+    <meta property="og:title" content="Bearer CLI: {{ title | escape }}"/>
+    <meta property="og:description" content="{{ meta.description | escape }}"/>
+    <meta property="og:image" content="/assets/img/bearer-cli-opengraph.png"/>
```
Comment (gotbadger) [inline docs/_src/_includes/layouts/base.njk]: Think you need to use the canonical url here ?

Comment (markmichon) [reply docs/_src/_includes/layouts/base.njk]: `og:url` is on 13, but I've gone ahead and added the canonical rel as well 👍

Comment (gotbadger) [reply docs/_src/_includes/layouts/base.njk]: I meant as in the full url so "https://docs.bearer.sh/assets/img/bearer-cli-opengraph.png"

Comment (markmichon) [reply docs/_src/_includes/layouts/base.njk]: Gotcha

**Review by gotbadger — APPROVED** [review]

**Pull request merged**